python/src/aac/__init__.py

```diff
@@ -1,5 +1,14 @@
 """The Architecture-as-Code tool."""
 
+# Run Python3.9 Check before continuing
+import sys
+
+if sys.version_info < (3, 9):
+    minor = sys.version_info.minor
+    major = sys.version_info.major
+    print(f"Python version {major}.{minor} is too low; AaC requires at least Python version 3.9 or higher to run.")
```

Comment:
Should we log this instead of (or in addition to) printing?

Reply:
The log configuration is actually one of the components that throws errors in sub 3.9 environments. I'm sure that we could configure the logger in a backward-compatible method, but I think print is sufficient for this use case.

Reply:
Ah, okay.